app/javascript/portal/components/SearchSuggestions.vue

```diff
@@ -27,7 +27,7 @@
           :href="generateArticleUrl(article)"
           class="flex-auto truncate text-base font-medium leading-6 w-full hover:underline"
         >
-          {{ article.title }}
```

Comment:
Is this an intentional change?

Reply:
@nithindavid I was checking if this change affects anywhere in the UI, and it's not reflecting. As mentioned in the linear I am not able to change the routes here for public articles redirect.